CONTRIBUTING.md

```diff
@@ -102,8 +102,10 @@ working on it.
      pipenv run python -m pre-commit install
    ```
 
-4. Make the change and create a
-   [pull request from your fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork).
+4. Make the change. You can also add your GitHub login in a new line of the `contributors.txt` file. We are using
```

Comment:
=>
4. Make the changes.&lt;br/&gt;
&nbsp;&nbsp;&nbsp;You may want to also add your GitHub login as a new line of the \`contributors.txt\` file located at the root
&nbsp;&nbsp;&nbsp;of this repository. We are using it to list our contributors in the Taipy documentation (see the "Contributing > Contributors" section) and thank them.